sdk/batch/Microsoft.Azure.Batch/CHANGELOG.md

```diff
@@ -1,16 +1,24 @@
 # Release History
 
-## 15.2.0-beta.1 (Unreleased)
+## 15.3.0 (2022-01-01)
```

Comment:
We may want to bump this to the current date (this date is the rough release date, not the API version)